test/helpers.js

```diff
@@ -73,9 +73,7 @@ var errorHandler = module.exports.errorHandler = function () {
 module.exports.createServer = function (initArgs, options, callback) {
   var app = require('connect')();
   var serverInit = function (middleware) {
-    var handler = options.handler || function (req, res) {
-      res.end('OK');
-    };
+    var handler = options.handler || function (req, res) { };
```

Comment:
After upgrade, this `res.end('OK')` was causing issues with the tests that headers were already sent.  I noticed most tests explicitly sent a response, but a handful relied on this default "OK", so I decided to remove it here, and explicitly add it to those tests.